src/Helpers/JsonEncoder.php

```diff
@@ -0,0 +1,42 @@
+<?php
```

Comment:
Shouldn't we use the library @Jalle19 made for this?

Reply:
@crisu83 we are, but we need to override the encoder to get rid of those unwanted characters

Reply:
Fair enough.